### PR TITLE
adding your task workflow

### PR DESCRIPTION
**Pull request by vdemeester**
- docs: Landing Page (README.md)
- Add a workflow to provide your task and pipeline documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,149 +1,109 @@
...
-
-Some annotations to look for are: 
-```
-metadata:
-  annotations:
-    pipelinesascode.tekton.dev/on-event: "[push]"
-    pipelinesascode.tekton.dev/on-target-branch: "[main]"
-    pipelinesascode.tekton.dev/max-keep-runs: "5"
-```
+# Contributing
```
Comment (koustavsaha) [inline README.md:88]: We should link https://github.com/openshift-pipelines/tektoncd-catalog/pull/66/files#diff-f9e4af9b0893ffa7261612dc76313ee6c8eb695632ff1003623f3c083fb30f31 this here

Comment (vdemeester) [reply README.md:88]: It is here : https://github.com/openshift-pipelines/tektoncd-catalog/pull/66/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R94. Maybe it's not super obvious ?

Comment (koustavsaha) [reply README.md:88]: We should move it up a little. Just below contributions section

**Pull request merged**